gui/default/syncthing/core/syncthingController.js

```diff
@@ -810,7 +811,7 @@ angular.module('syncthing.core')
         };
 
         $scope.deviceName = function (deviceCfg) {
-            if (typeof deviceCfg === 'undefined') {
+            if (typeof deviceCfg === 'undefined' || deviceCfg.deviceID === undefined) {
```

Comment:
`/me puts on his nit picking hat`

Can we at least check for undefinedness in the same way twice in the same `if`? :)